web/src/features/panels/zone/ZoneDetails.tsx

```diff
@@ -190,3 +193,29 @@ function ZoneDetailsContent({
 
   return children as JSX.Element;
 }
+
+const useScrollHashIntoView = (isLoading: boolean) => {
+  const { hash, pathname, search } = useLocation();
+  const navigate = useNavigate();
+  useEffect(() => {
+    if (isLoading) {
+      return;
+    }
+
+    if (hash) {
```

Comment:
```suggestion
    if (hash && hash in Charts) {
```
Will make it handle weird cases like /zone/dk-dk2######

Reply:
Great point!